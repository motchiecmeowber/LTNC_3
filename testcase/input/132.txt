INSERT x number
INSERT y string
BEGIN
INSERT x string
INSERT z number
ASSIGN z 100
LOOKUP z
END
PRINT