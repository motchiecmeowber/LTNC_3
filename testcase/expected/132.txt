success
success
success
success
success
1
x//0 y//0